config/src/main/java/io/confluent/common/config/ConfigDef.java

```diff
@@ -361,7 +361,11 @@ else if (value instanceof String)
           if (value instanceof Class) {
             return (Class<?>) value;
           } else if (value instanceof String) {
-            return Class.forName(trimmed);
+            if (Thread.currentThread().getContextClassLoader() != null) {
```

Comment:
This should certainly fix the issue. But I was wondering if it made more sense to symlink the confluent-common into kafka-serde-tools. This would mean that confluent-common is available as part of the plugin-path and hence same class loader as serializers & converters. This should also be fixed in our packaging.

Reply:
Yes, that is an alternative solution. We can create a softlink to confluent-common wherever we create links to storage-common. I just checked it, and that fixes the problem too.

Reply:
@kkonstantine will be great to hear your thoughts on this before we purse the above solution.